swarm_listen.go

```diff
@@ -111,7 +111,7 @@ func (s *Swarm) AddListenAddr(a ma.Multiaddr) error {
 					// ignore.
 					return
 				default:
-					log.Warningf("add conn %s failed: ", err)
+					log.Warnf("add conn %s failed: ", err)
```

Comment:
If we're changing these, we might as well switch to a structured version:

```go
log.Warnw("adding connection failed", "to", p, "error", err)
```

Reply:
Fixed in #255.